src/types.h

```diff
@@ -113,3 +113,11 @@ typedef struct {
     request_type_e req_type;  /// user request
     bip32_path_t bip32_path;
 } global_ctx_t;
+
+/**
+ * Internal storage structure
+ */
+
+typedef struct internalStorage_t {
+    uint32_t secret;
```

Comment:
The code looks ok, but I feel I'm lacking some context to fully understand the motivation for the PR.

What will this secret be used for exactly in the future?

Reply:
Ok, I've been informed that this is related to https://github.com/HathorNetwork/hathor-ledger-app/issues/28